src/runtime/containerd-shim-v2/create.go

```diff
@@ -259,3 +272,80 @@ func doMount(mounts []*containerd_types.Mount, rootfs string) error {
 	}
 	return nil
 }
+
+func configureNonRootHypervisor(runtimeConfig *oci.RuntimeConfig) error {
+	userName, err := createVmmUser()
```

Comment:
I'm afraid, if something fails, you'll end up leaking a lot of users into the system, maybe you should add a `defer` to check the error

```go
defer func() {
    if err != nil {
        removeVmmUser()
    }
}()